demo/src/screens/MainScreen.js

```diff
@@ -413,19 +413,19 @@ const SectionItem = React.memo(props => {
       activeOpacity={1}
       style={Dividers.d10}
     >
-      <Text style={[item.deprecate && styles.entryTextDeprecated]} grey10 text80>
+      <Text style={[item.deprecate && styles.entryTextDeprecated]} text80>
         {item.title}
       </Text>
-      <Icon source={chevronIcon} style={{tintColor: Colors.grey10}} supportRTL/>
+      <Icon source={chevronIcon} style={{tintColor: Colors.$iconDefault}} supportRTL/>
     </TouchableOpacity>
   );
 });
 
 const SectionHeader = React.memo(props => {
   const {section} = props;
   return (
-    <View backgroundColor={'white'}>
-      <Text back marginV-20 marginH-20 text60M>
+    <View bg-$backgroundDefault>
```

Comment:
Consider removing the background and keeping it transparent so it'll take the background from the view behind it.

Reply:
The reason I didn't removed the background is because if the background will be transparent you will see the Item beneath the header, the header will override the item title and you will see 2 text one override the other one.
It should stay with background color.